src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -29,14 +30,14 @@ const FILE_TYPES = [
   'jpg',
   'bmp'
 ];
+
 const MIN_SIZE = 350;
+const MIN_RATIO = 0.2;
+const MAX_RATIO = 1.7;
+const WARN_RATIO = 1.3;
 
-function detectDragAndDrop() {
-  const div = document.createElement('div');
-  const supportsDragAndDrop = ('draggable' in div) || ('ondragstart' in div && 'ondrop' in div);
-  const iOS = !!navigator.userAgent.match('iPhone OS') || !!navigator.userAgent.match('iPad');
-  const dragAndDropDetected = supportsDragAndDrop && window.FileReader;
-  return dragAndDropDetected && !iOS;
+function isSmallScreen(width) {
+  return  width < 1201;
```

Comment:
Could make a constant for this number, too.

Reply:
Thanks @jbalboni, great point, changed.